RELEASE_NOTES.md

```diff
@@ -17,22 +17,43 @@ OPAE\'s goal is to accelerate FPGA adoption. It is a community effort to simplif
 More information about OPAE can be found
 at http://01.org/OPAE.
 
-Open Programmable Acceleration Engine (OPAE) 1.3.0 Release Notes
+Open Programmable Acceleration Engine (OPAE) 1.4.0 Release Notes
 -----------------------------------------------------------------
 
 This document provides the Release Notes for the Open Programmable
-Acceleration Engine (OPAE) 1.3.0 release.
+Acceleration Engine (OPAE) 1.4.0 release.
 
 System Compatibility
 --------------------
 
 -   Hardware: tightly coupled FPGA products and programmable FPGA
     acceleration cards for Intel(R) Xeon(R) processors:
-    - Intel(R) PAC with Arria(R) 10 GX FPGA (PCI ID: 0x09c4) FIM version 1.0.3 (1.0 Production)
+    - Intel(R) PAC with Arria(R) 10 GX FPGA (PCI ID: 0x09c4) FIM version 1.1.2-1 (1.2 Production)
     - Intel(R) Xeon with Integrated FPGA (PCI ID: 0xbcc0) FIM version 6.4.0
+    - Intel® FPGA Programmable Acceleration Card N3000 (PCI ID: 0x0b30) FIM version D.1.0.13 (1.0 Production)
 
 -   Operating System: tested on Red Hat Enterprise Linux 7.3 and 7.4, Ubuntu 16.04, 
-    SUSE SLE 12 SP3 and CentOS 7.4, with Linux kernels 3.10 through 4.7
+    SUSE SLE 12 SP3 and CentOS 7.4, with Linux kernels 3.10 through 4.19
```

Comment:
Should SuSE be listed (I don't know)
Listing the RedHat kernel but not the SuSE kernel seems confusing to me.
And this should be CentOS 7.6 (not 7.4), right?
So maybe something like: SUSE SLE 12 SP3 and CentOS 7.4 (with both the CentOS 7.4 kernel and the community 4.19 LTS kernel)